packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+function Term({
+  label,
+  after = '',
+  endAlign,
+  small = true,
+  children,
+  cx,
+  styles,
+}: {
+  label: string;
+  after?: React.ReactNode;
+  endAlign?: boolean;
+  small?: boolean;
+  keyTextRegular?: boolean;
+  children: NonNullable<React.ReactNode>;
+} & WithStylesProps) {
+  return (
+    <>
+      <dt>
+        <Row after={endAlign && after}>
+          <Text inline small={small}>
+            {label}
+          </Text>
+          {!endAlign && (
+            <Spacing inline left={1}>
+              {after!}
+            </Spacing>
+          )}
+        </Row>
+      </dt>
+      <dd className={cx(styles.value)}>
+        <Text>{children}</Text>
+      </dd>
+    </>
+  );
+}
+
+export default withStyles(() => ({
+  value: {
+    // reset styles
+    marginInlineStart: 0,
```

Comment:
you can do this reset in index.tsx - which would help keep all the dt/dd styling in one place